components/02-elements/form/input/_input.scss

```diff
@@ -1,32 +1,33 @@
-$input__padding                      : 0 $spacer--medium !default;
+$input__field-padding                : 0 $spacer--medium !default;
+$input__field-padding--textarea      : $spacer--medium !default;
 $input__field-spacing                : 40px !default;
 $input__field-border                 : 1px solid $form-elements-border-color !default;
 $input__field-border-radius          : $form-elements-radius !default;
 $input__field-border-radius--textarea: $form-elements-radius--small !default;
 $input__field-line-height--textarea  : $font-line-height !default;
+$input__margin-bottom                : $spacer !default;
 $input__placeholder-color            : $gray !default;
-
-$input__min-height--textarea: 100px !default;
+$input__label-margin-right           : $spacer--medium !default;
+$input__min-height--textarea         : 100px !default;
 
 .input {
-    margin-bottom: $spacer;
+    margin-bottom: $input__margin-bottom;
 
     &--inline {
         display: flex;
         flex-flow: row nowrap;
         align-items: center;
-
-        .input__label {
-            margin-bottom: 0;
-            margin-right: $spacer--medium;
-        }
+    }
+    &__label {
```

Comment:
That's not a default styles of the label for input, it should be added only to the `inline` modificator.

Reply:
moved